### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by phiiil69** [xtekky/gpt4free#2311] [General]
How can we be sure that our private OpenAI API won’t experience a leak or be used fraudulently?
Best regards,
Phil

**Comment by Lamparter** [comment]
@phiiil69 Hi there! This repository is not owned by OpenAI and is designed to allow you to access AI without an API key. If you'd like to contact OpenAI for a question, you can do so [here](https://help.openai.com/).

###### If you found this useful, feel free to mark it as an answer!
